src/data/pokemon.csv

```diff
@@ -1 +1 @@

Porygon,normal,100,240,125,175,145,155,85,Signal-beam,75,15,bug,special,confusion,0,Gravity,0,5,psychic,status,none,0,Skull-bash,130,10,normal,physical,none,0,Teleport,0,20,psychic,status,none,0

Omanyte,rock,100,180,85,185,205,115,75,Supersonic,0,20,normal,status,confusion,0,Bite,60,25,dark,physical,none,0,Sleep-talk,0,10,normal,status,none,0,Confide,0,20,normal,status,none,0

Omastar,rock,100,250,125,235,255,145,115,Stone-edge,100,5,rock,physical,none,0,Submission,80,20,fighting,physical,none,0,Spike-cannon,20,15,normal,physical,none,0,Withdraw,0,40,water,status,none,0

Kabuto,rock,100,170,165,115,185,95,115,Aurora-beam,65,20,ice,special,none,0,Secret-power,70,20,normal,physical,none,0,Confide,0,20,normal,status,none,0,Reflect,0,20,psychic,status,none,0

Kabutops,rock,100,230,235,135,215,145,165,Return,0,20,normal,physical,none,0,Snore,50,15,normal,special,none,0,Blizzard,110,5,ice,special,freeze,0,Ancient-power,60,5,rock,special,none,0

Aerodactyl,rock,100,270,215,125,135,155,265,Bulldoze,60,20,ground,physical,none,0,Agility,0,30,psychic,status,none,0,Dragon-rage,0,10,dragon,special,none,0,Roost,0,10,flying,status,none,50

Snorlax,normal,100,430,225,135,135,225,65,Round,60,15,normal,special,none,0,Power-up-punch,40,20,fighting,physical,none,0,Bubble-beam,65,20,water,special,none,0,Thunderbolt,90,15,electric,special,paralysis,0

Articuno,ice,100,290,175,195,205,255,175,Sandstorm,0,10,rock,status,none,0,Aurora-veil,0,20,ice,status,none,0,Ancient-power,60,5,rock,special,none,0,Bubble-beam,65,20,water,special,none,0

Zapdos,electric,100,290,185,255,175,185,205,Magnetic-flux,0,20,electric,status,none,0,Reflect,0,20,psychic,status,none,0,Twister,40,20,dragon,special,none,0,Whirlwind,0,20,normal,status,none,0

Moltres,fire,100,290,205,255,185,175,185,Rage,20,20,normal,physical,none,0,Incinerate,60,15,fire,special,none,0,Hidden-power,60,15,normal,special,none,0,Hyper-beam,150,5,normal,special,none,0

Dratini,dragon,100,192,133,105,95,105,105,Slam,80,20,normal,physical,none,0,Confide,0,20,normal,status,none,0,Facade,70,20,normal,physical,none,0,Iron-tail,100,15,steel,physical,none,0

Dragonair,dragon,100,232,173,145,135,145,145,Water-pulse,60,20,water,special,confusion,0,Skull-bash,130,10,normal,physical,none,0,Iron-tail,100,15,steel,physical,none,0,Rest,0,10,psychic,status,none,0

Dragonite,dragon,100,292,273,205,195,205,165,Fly,90,15,flying,physical,none,0,Brutal-swing,60,20,dark,physical,none,0,Superpower,120,5,fighting,physical,none,0,Tailwind,0,15,flying,status,none,0

Mewtwo,psychic,100,322,225,313,185,185,265,Thunder,110,10,electric,special,paralysis,0,Strength,80,15,normal,physical,none,0,Double-edge,120,15,normal,physical,none,0,Calm-mind,0,20,psychic,status,none,0

Mew,psychic,100,310,205,205,205,205,205,Defog,0,15,flying,status,none,0,Return,0,20,normal,physical,none,0,Water-gun,40,25,water,special,none,0,Bide,0,10,normal,physical,none,0


```

Comment:
Make sure to reset Charizard's and all other Pokemon's moves to non-crazy values